README.md

```diff
@@ -16,8 +41,25 @@ defines your hosts and maps them to host groups declared in
     [logging_hosts]
     server2
 
-Put any necessary configuration into a ansible variables files.  For
-example, I have a file called `config.yml` that contains:
+There are two high-level groups that can be used to control service
+placement:
+
+- `am_hosts`: The playbooks will install availability monitoring
+  software (Sensu, Uchiwa, and support services) on these hosts.
+
+- `logging_hosts`: The playbooks will install the centralized logging
+  stack (Elasticsearch, Kibana, Fluentd) on these hosts.
+
```

Comment:
We are not going to document pm_hosts? I mean performance monitoring hosts deployment should be functional right? It does not matter we don't have analogical patch in TripleO, does it?